python/pyarrow/_flight.pyx

```diff
@@ -919,20 +1002,30 @@ cdef class FlightServerBase:
     cdef:
         unique_ptr[PyFlightServer] server
 
-    def run(self, port, auth_handler=None):
+    def run(self, location, auth_handler=None, **kwargs):
```

Comment:
Writing `tls_cert_chain=None, tls_private_key=None` instead will make the signature more informative (and also avoid typos in argument names).